### ISSUE TITLE
UI usability improvement

### ISSUE THREAD

**Issue by tkbletsc** [Salandora/octoprint-customControl#5]
First, thanks for this plugin. It's going to allow me to use OctoPrint to run my CNC machine by filling the UI gaps in what you need to do for CNC that you don't need for 3D printing.

That said, I almost gave up on the plugin because it appeared to me to be broken -- when I went to "Custom Controls" in settings for the first time, all I saw was the green outline box. It wasn't until I noticed a screenshot of a context menu on the plugin page that I realized that it did work properly, and I just needed to right-click to get started.

At minimum, I think it would be nice if at the top of the UI it said something like "to add containers and controls, right-click the green box", or better yet, since the only option in that context menu is to make a container, why not just replace the green box with a "create container" button?

**Comment by ulope** [comment]
Yes, the same happened to me. I actually uninstalled it because I thought it didn't work correctly and only by chance saw how it's supposed to work in some random youtube video.

**Comment by Zblay** [comment]
Totally agree, as a matter of fact, i create custom controls about a year ago, and needed new ones recently, and i thought the plugin brok, because i didn't remember the green box thing.

The UI needs (very simple) improvements.

**Comment by ahernsean** [comment]
I loaded this up on my server and saw the green box. But I'm on an iPad! There's no right click! I would think that just a "+" button could be used instead. That would be simple and intuitive.
